<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Averaging</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>920</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>1000</height>
    </size>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit_OutputConsole">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>100</y>
       <width>491</width>
       <height>150</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelHeader_OutputConsole">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>70</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>75</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Messages&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_ClearAll">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>170</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Value_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteButton_1">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>130</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton_Values_Errors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;span&gt;&amp;#177;&lt;/span&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Error_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_Value">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_Uncertainty">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Uncertainty</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteButton_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Value_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Error_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;span&gt;&amp;#177;&lt;/span&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteButton_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>210</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Value_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;span&gt;&amp;#177;&lt;/span&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Error_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_CalculateAverage">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>150</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calculate Average</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_SaveResults">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>310</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
